src/main/java/tech/illuin/pipeline/commons/Reflection.java

```diff
@@ -15,6 +17,21 @@
  */
 public final class Reflection
 {
+    private static final Map<Class<?>, Class<?>> PRIMITIVE_MAP;
+
+    static {
+        PRIMITIVE_MAP = new HashMap<>();
```

Comment:
Why not a `Map.of(...)` here ?

Reply:
Initially because of the size of the map (`Map.of` doesn't work with varargs due to the alternance between keys and values), but on the other hand if it fits it's not like we'll get new types of primitives every other day.

I like the idea of the map being immutable so, I'll make the change 👍